examples/sycl/pvc/flash_attention_v2/online_softmax.hpp

```diff
@@ -48,8 +48,6 @@
   }
 #endif
 
-#define EXP sycl::native::exp
-#define DIV sycl::native::divide
 // TODO:: Temporary using OpenCL function as sycl_reduce_over_group spills on sycl::maximum<>() operation
```

Comment:
I don't observe this problem. When I change the code to use `reduce_over_group(g, maxptr, sycl::maximum<>());`. Do you still observe the problem?

Reply:
No I dont observe the spill issue anymore, So I have added the SYCL version instead